modules/claims_api/app/controllers/concerns/claims_api/mpi_verification.rb

```diff
@@ -5,25 +5,8 @@ module MPIVerification
     extend ActiveSupport::Concern
 
     included do
-      before_action :verify_mpi
-
       def verify_mpi
-        raise 'MPI user not found' unless target_veteran.mpi_record?
-      rescue => e
-        log_message_to_sentry('MPIError in claims',
-                              :warning,
-                              body: inspected_error || e.message)
-
-        render json: { errors: [{ status: 400, detail: 'Not enough Veteran information, functionality limited.' }] },
-               status: :bad_request
-      end
-
-      private
-
-      def inspected_error
-        target_veteran.mpi&.response&.error&.inspect
-      rescue
-        nil
+        raise ::Common::Exceptions::ParameterMissing, 'MPI user' unless target_veteran.mpi_record?
```

Comment:
Any value in keeping some of the "_Not enough Veteran information, functionality limited_" context?
I just wonder if a 400 with a message of "MPI User" is going to be enough to let a consumer know that they haven't supplied enough veteran info.